tests/spdy-over-ws-test.js

```diff
@@ -62,6 +67,43 @@ describe('spdy', () => {
     conn.end()
   })
 
-  it('open a multiplex stream from dialer and write to it', (done) => {})
-  it('open a multiplex stream from listener and write to it', (done) => {})
+  it('open a multiplex stream from dialer and write to it', (done) => {
+    listener.once('stream', (conn) => {
+      conn.pipe(conn)
+    })
+
+    const conn = dialer.newStream()
+    conn.write('hey')
+    conn.write('ho')
+    conn.end()
+
+    conn.on('error', (err) => {
+      expect(err).to.not.exist
+    })
+    conn.pipe(bl((err, data) => {
+      expect(err).to.not.exist
+      expect(data.toString()).to.equal('heyho')
+      done()
+    }))
+  })
+
+  it('open a multiplex stream from listener and write to it', (done) => {
+    dialer.once('stream', (conn) => {
+      conn.pipe(conn)
+    })
+
+    const conn = listener.newStream()
+    conn.write('hey')
+    conn.write('ho')
```

Comment:
@xicombd this is not enough, they just get buffered and written on the same packet. It needs to be more stress tested